### ISSUE TITLE
some questions about train.py

### ISSUE THREAD

**Issue by dr18112004** [zhanghang1989/PyTorch-Encoding#287]
Hi, thanks for your work. My environment is:
ubuntu==18.04
python==3.6.8
pytorch==1.4.0
cuda==10.0
when I first run the train.py, there is a error about 'ninja'. I have resolved it and obtained a result. But when I change the model from 'encnet' to 'deeplab', the code doesn't react at all, and there are no errors. Then I reuse the 'encnet', the codedoesn't react at all, and there are no errors. How can I resolve it?

**Comment by zhanghang1989** [comment]
That sounds wired.
These are the best practices for setting up the environment https://hangzhang.org/PyTorch-Encoding/notes/compile.html#detailed-steps

**Comment by dr18112004** [comment]
Thank you very much for your quick reply，I will try the practices. Thanks again

**Comment by dr18112004** [comment]
Hi, I have tried the practices following your provided, and loading the data are fine, but the code can't begin to train, do you have any suggestions to solve this problem. Thanks again.

the problem as follows:
"
Using poly LR scheduler with warm-up epochs of 0!
Starting Epoch: 0
Total Epoches: 200
  0%|                                                                      | 0/1384 [00:00<?, ?it/s]
=>Epoch 0, learning rate = 0.0001,                     previous best = 0.0000
"
and the code can't begin to train.

**Comment by zhanghang1989** [comment]
That's wired. Have you tried `train_dist.py`?

**Comment by dr18112004** [comment]
I reduce the batch_size==16, and the code is able to train properly .  Thanks again.
